README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Live Demo
 
-[Math Magicians App](https://joshuaivie.github.io/BookStore)
+[Book Store App](https://joshuaivie.github.io/BookStore)
```

Comment:
- [ ] (Optional) The link is broken. When I click on it, it shows a _**404**_ error. Kindly deploy it properly or remove it from the `README.md` file. Doing this will make your work `README.md` file look very professional.